core/Tracker/Visit.php

```diff
@@ -580,7 +580,7 @@ private function getVisitStandardLength()
      */
     private function setIdVisitorForExistingVisit($valuesToUpdate)
     {
-        if (strlen($this->visitProperties->getProperty('idvisitor')) == Tracker::LENGTH_BINARY_ID) {
+        if (strlen($this->visitProperties->getProperty('idvisitor') ?? '') == Tracker::LENGTH_BINARY_ID) {
             $valuesToUpdate['idvisitor'] = $this->visitProperties->getProperty('idvisitor');
         }
```

Comment:
Let's do it in a similar way how it's done in `getVisitorIdcookie` method then
```suggestion
        $idVisitor = $this->visitProperties->getProperty('idvisitor');
        if (!empty($idVisitor) && Tracker::LENGTH_BINARY_ID == strlen($idVisitor)) {
            $valuesToUpdate['idvisitor'] = $idVisitor;
        }
```